ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -589,6 +596,11 @@ VALUE allocate(VALUE klass) {
     context_info->context = new Persistent<Context>();
     context_info->context->Reset(context_info->isolate, context);
 
+    if (rb_funcall(rb_cObject, rb_intern("const_defined?"), 1, rb_str_new2("DateTime")) == Qtrue)
```

Comment:
minor change ... check if it is already set, set rb_cDateTime to NULL by default, only do the funcall if its NULL